packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -180,6 +180,66 @@ public void onScanCompleted(String path, Uri uri) {
     this.fileUtils = fileUtils;
   }
 
+  // -- Video --
+  // Choose Video From Gallery
```

Comment:
These comments provide very little information on top of the name of the method.